gmusicapi/clients/musicmanager.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
-from future import standard_library
-standard_library.install_aliases()
+from future.utils import PY3
 from past.builtins import basestring
 from builtins import *  # noqa
+
 import os
 from socket import gethostname
 import time
-import urllib.request
-import urllib.parse
-import urllib.error
 from uuid import getnode as getmac
 import webbrowser
 
+if PY3:
```

Comment:
Yeah, Python 2 support should definitely be dropped by the time Python 4 comes around. 3.7 isn't even slated for release until April 2018, which is actually a shorter release cycle from 3.6 release date than normal.

And, as another example, Flask actually uses ``if not PY2`` in their compat layer which could also cause issues on a new major version.